chain/actors/actor_miner2.go

```diff
@@ -404,6 +407,19 @@ func (sma StorageMinerActor2) GetPower(act *types.Actor, vmctx types.VMContext,
 	return self.Power.Bytes(), nil
 }
 
+func (sma StorageMinerActor2) GetPower2(act *types.Actor, vmctx types.VMContext, params *struct{}) ([]byte, ActorError) {
+	_, self, err := loadState(vmctx)
+	if err != nil {
+		return nil, err
+	}
+
+	if self.SlashedAt != 0 {
```

Comment:
TODO: check if there's anything that calls this that assumes real power even if slashed (some state api methods do that iirc)